sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubAsyncClient.java

```diff
@@ -142,4 +145,13 @@ EventHubConsumerAsyncClient createConsumer(String consumerGroup, int prefetchCou
     public void close() {
         connectionProcessor.dispose();
     }
+
+    /**
+     * Gets the client identifier.
+     *
+     * @return The unique identifier string for current client.
+     */
+    public String getIdentifier() {
```

Comment:
Do other languages expose this?

Reply:
They should be.    .NET exposes it on all clients, JS and Python are not currently exposing it, but I'll make sure there's an issue to track getting it added.